### PR TITLE
Fixed concatenation error

### PR DESCRIPTION
**Pull request by sunlin-link**

### CODE REVIEW

**Comment by sunlin-link** [comment]
Fixes https://github.com/intel/webml-polyfill/issues/58

File: services/ml/compilation_impl_mac.mm [hunk 1]
```diff
@@ -250,7 +250,6 @@ void ComputeBNNSOffsetForImplicitPadding(
     memory_size_ = model->memory_size_;
     memory_.reset(new int8_t[memory_size_]);
     memcpy(memory_.get(), model->memory_.get(), memory_size_);
-    is_bnns_ = true;
```
Comment (fujunwei) [inline services/ml/compilation_impl_mac.mm]: Maybe @lisa0314 need take a look.

File: services/ml/compilation_impl_mac.mm [hunk 2]
```diff
@@ -1065,15 +1059,12 @@ void ComputeBNNSOffsetForImplicitPadding(
 
     auto impl =
         std::make_unique<ExecutionImplMac>(this, std::move(memory_handle));
-    if (!impl->IsValid()) {
-      std::move(callback).Run(mojom::BAD_DATA, std::move(init_params));
```
Comment (fujunwei) [inline services/ml/compilation_impl_mac.mm]: What's the result if return in here?

File: services/ml/compilation_impl_mac.mm [hunk 3]
```diff
@@ -981,11 +981,6 @@ void ComputeBNNSOffsetForImplicitPadding(
         getScalarInt32(values_[inputs[inputs.size() - 1]], memory_.get());
     LOG(ERROR) << "*****axis******: " << axis;
 
-    if (axis != 3) {
```
Comment (huningxin) [inline services/ml/compilation_impl_mac.mm]: Do we have code to handle cases where `axis` is not 3?

**Pull request closed without merging**